receiver/kafkametricsreceiver/metadata.yaml

```diff
@@ -0,0 +1,76 @@
+name: kafkametricsreceiver
+
+labels:
+  topic:
+    description: The ID (integer) of a topic
+  partition:
+    description: The number (integer) of the partition
+  group:
+    description: The ID (string) of a consumer group
+
+metrics:
+  brokers.count:
```

Comment:
I think we should probably prefix all these with `kafka.` since they're kafka-specific. Also make sure you commit the generated go file.

Reply:
`generated_metrics.go` is committed above - it's collapsed by default because it's a bit long